<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="lblService">
     <property name="text">
      <string>Код услуги</string>
     </property>
     <property name="buddy">
      <cstring>cmbService</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="lblAmount">
     <property name="text">
      <string>Количество</string>
     </property>
     <property name="buddy">
      <cstring>edtAmount</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="lblSum">
     <property name="text">
      <string>Сумма</string>
     </property>
     <property name="buddy">
      <cstring>edtSum</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QDoubleSpinBox" name="edtSum">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>9999999.990000000223517</double>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDoubleSpinBox" name="edtAmount">
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="CRBServiceComboBox" name="cmbService"/>
   </item>
   <item row="6" column="4">
    <widget class="QDoubleSpinBox" name="edtUET">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QLabel" name="lblUET">
     <property name="text">
      <string>УЕТ</string>
     </property>
     <property name="buddy">
      <cstring>edtUET</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="CRBComboBox" name="cmbUnit"/>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="lblUnit">
     <property name="text">
      <string>Единицы учета</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblPrice">
     <property name="text">
      <string>Цена по тарифу</string>
     </property>
     <property name="buddy">
      <cstring>edtPrice</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QDoubleSpinBox" name="edtPrice">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>9999999.990000000223517</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblServiceNameSource">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="QLabel" name="lblServiceName">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblAccount">
       <property name="text">
        <string>Переместить в счет</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbAccount">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBServiceComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbservicecombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
